.backportrc.json

```diff
@@ -0,0 +1,15 @@
+{
+  "upstream": "elastic/elasticsearch",
+  "targetBranchChoices": [
+    { "name": "master", "checked": true },
+    { "name": "7.x", "checked": true },
+    "7.13",
+    "6.8"
+  ],
+  "targetPRLabels": ["backport"],
+  "branchLabelMapping": {
+    "^v8.0.0$": "master",
+    "^v7.14.0$": "7.x",
+    "^v(\\d+).(\\d+).\\d+$": "$1.$2"
```

Comment:
What happens when we start prepping the 8.0 release, and we (I assume) have tags like `v8.0.0-rc1` etc? Do we need to accommodate that at all? I'm struggling to picture what the workflow looks like in that situation.